### PR TITLE
Support custom display id inside docker container

### PR DESCRIPTION
**Pull request by akekawit**
Currently, the XVNC capability mounts the entire `X_SOCKET_PATH` directory and sets the DISPLAY environment variable to appropriate display id. However, some devices are hardcoded to use :0 display. This commit updates the capability to accept docker_display_id and only mounts the allocated display id to `<X_SOCKET_PATH>/X<docker_display_id>` on Docker. If docker_display_id is not provided, fall back to using the same display id.

### CODE REVIEW

File: cirque/capabilities/xvnccapability.py [hunk 1]
```diff
@@ -24,16 +24,19 @@
 class XvncCapability(BaseCapability):
     X_SOCKET_PATH = '/tmp/.X11-unix'
 
-    def __init__(self, localhost=True, display_id=0):
+    def __init__(self, localhost=True, display_id=0, docker_display_id=None):
```
Comment (mingchik) [inline cirque/capabilities/xvnccapability.py:27]: why do we need add this new docker_display_id parameter?
can we not just using display_id in docker?

Comment (akekawit) [reply cirque/capabilities/xvnccapability.py:27]: For our use case, the display id inside docker container always has to be 0. If we only use display_id, it means display id 0 has to be always available which might not be the case if we are bringing up more than 1 device.

**Review by mingchik — APPROVED** [review]

**Comment by yunhanw-google** [comment]
Could you add one unit test for it? cirque/capabilities/test/test_xvnc_capability.py?
Some example is here
https://github.com/openweave/cirque/commit/8c06982f4c4a2133ea6713e1acd631786c3b39b6
thanks

**Comment by akekawit** [comment]
> Could you add one unit test for it? cirque/capabilities/test/test_xvnc_capability.py?
> Some example is here
> [8c06982](https://github.com/openweave/cirque/commit/8c06982f4c4a2133ea6713e1acd631786c3b39b6)
> thanks

Added unit tests

**Review by yunhanw-google — APPROVED** [review]

**Pull request merged**